js/models/localattachment.js

```diff
@@ -0,0 +1,33 @@
+ /**
+  * Mail
+  *
+  * This file is licensed under the Affero General Public License version 3 or
+  * later. See the COPYING file.
+  *
+  * @author Christoph Wurst <christoph@winzerhof-wurst.at>
+  * @author Luc Calaresu <dev@calaresu.com>
+  */
+
+define(function(require) {
+	'use strict';
+
+	var Attachment = require('models/attachment');
+
+	var LocalAttachment = Attachment.extend({
+		defaults: {
+			progress: 0,
+			uploadStatus: 0  /* 0=pending, 1=ongoing, 2=error, 3=success */
+		},
+		initialize: function() {
```

Comment:
unused. remove this and then the parent constructor will be called automatically for you.